web/portal/platform/src/entities/Brand/Field/PostalCode.tsx

```diff
@@ -0,0 +1,18 @@
+import {
+  PropertyCustomFunctionComponent,
+  PropertyCustomFunctionComponentProps,
+} from '@irontec/ivoz-ui';
+import { BrandPropertyList } from '../BrandProperties';
+
+type RouteTypeValues = BrandPropertyList<string>;
+type RouteTypeProps = PropertyCustomFunctionComponent<
+  PropertyCustomFunctionComponentProps<RouteTypeValues>
+>;
+
+const PostalCode: RouteTypeProps = (props): JSX.Element | null => {
```

Comment:
In every custom component that doesn't return null:

```suggestion
const PostalCode: RouteTypeProps = (props): JSX.Element => {
```